yourgov/components/FormMobileFoodVendorPermit/FormState.ts

```diff
@@ -1,5 +1,19 @@
+import { type FieldErrors, type FieldValues } from 'react-hook-form';
 import { DeepPartial } from '../../lib/types';
-import { task1DefaultFormState, Task1FormState } from './FormTask1FormState';
+import {
+	task1DefaultFormState,
+	type Task1FormState,
+} from './task1/FormTask1FormState';
+import { type Task1StepNumber } from './task1/FormTask1Provider';
+
+export type Completion = {
+	completed: boolean;
+	started?: boolean;
+};
+
+interface TaskCompletion extends Completion {
+	completedRecently: boolean;
+}
```

Comment:
`interface` is something we've kept out of the DS, in favour of `type`. Extending `interface`s can get confusing compared with the more declaritive `type =` approach. I assume this is why we've done it historically. We also don't use any classes.

```suggestion
type TaskCompletion = Completion & {
	completedRecently: boolean;
}
```